src/wazuh_modules/vulnerability_scanner/CMakeLists.txt

```diff
@@ -55,6 +55,7 @@ foreach(schema IN LISTS Schemas)
       COMMAND ${SRC_FOLDER}/external/flatbuffers/build/flatc
       ARGS --cpp
       ARGS -o "${CMAKE_CURRENT_SOURCE_DIR}/include" "${FILE}"
+      ARGS --no-warnings
```

Comment:
We might be missing some important warnings, but it seems there isn't another option.

Reply:
I didn't find another option to suppress specific warning types